sites/main-site/src/content/hackathon-projects/hackathon-march-2025/clipseq2.0.md

```diff
@@ -2,6 +2,7 @@
 title: nf-core/clipseq v2.0.0!
 category: pipelines
 slack: https://nfcore.slack.com/archives/C01571V17PH
+image: /assets/images/events/2025/hackathon-march/bernie_clipseq_meme.png
```

Comment:
please upload the image and also add a descriptive alt text here in the frontmatter under `image_alt`.